### ISSUE TITLE
feature request: make it possible to just specify a chromosome in query

### ISSUE THREAD

**Issue by tommycarstensen** [slowkow/pytabix#4]
Current syntax is:
records = tb.querys("1:1000000-1250000")

It would be great, if I could just get all variants on chromosome 1 by doing one of these:
records = tb.querys("1")
records = tb.querys("1", 0, -1)

**Comment by slowkow** [comment]
I agree that this would be useful. I need to learn the specification of the tabix index file in order to implement this feature.

Sorry for the delayed response, but I'm always busy with other work. Patches are very welcome.

**Comment by johnwallace123** [comment]
I believe that this is possible using simply tb.querys("1").  It seems to work for me, and it's in line with what tabix provides.
